packages/blocks/src/api/children.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
@aduth I added the `children.js` and `node.js` files back because I see that some functions were exposed. I don't think any plugin will have used these apart from `concat`. I removed unused function, deprecated and mapped exposed functions, but left documentation intact. What's the best deprecation strategy here? Does this look good?